tests/unit/test-firefox/test.firefox.js

```diff
@@ -545,6 +646,136 @@ describe('firefox', () => {
 
   });
 
+  describe('defaultCreateProfileFinder', () => {
+
+    it('creates a finder', async () => {
+      const FxProfile = {
+        Finder: sinon.spy(() => () => Promise.resolve()),
+      };
+      firefox.defaultCreateProfileFinder({FxProfile});
+      sinon.assert.called(FxProfile.Finder);
+      sinon.assert.calledWith(
+        FxProfile.Finder,
+        sinon.notOk,
```

Comment:
This `sinon.assert.calledwith(FxProfile.Finder, sinon.notOk);` assertion doesn't seem right (or it is not very readable), shouldn't it be something like the following?

```js
firefox.defaultCreateProfileFinder({FxProfile});
sinon.assert.calledWith(FxProfile.Finder, sinon.match(undefined));
```